frontend/apps/quarano-frontend/src/assets/i18n/en.json

```diff
@@ -366,5 +366,13 @@
 		"ZUM_TAGEBUCH": "To the diary",
 		"ZU_DEN_INDEXFÄLLEN": "To the index cases",
 		"ZU_DEN_KONTAKTPERSONEN": "To the contacts"
-	}
+	},
+  "CASE_DETAILS": {
```

Comment:
Kannst du den Schlüssel bitte auf den Namen der Komponente beziehen, auf den er sich bezieht? Das wäre in dem Fall glaub ich DIARY_ENTRIES_LIST_ITEM

Reply:
mir ist gerade aufgefallen, dass du in deinem Diary-PR Texte für das GA-Frontend übersetzt. Eigentlich sind die GA-Seiten alle auf Deutsch. Die Übersetzung ist bisher nur für den Bürger... Lass es aber erst mal drin, evtl. kommt das ja noch...